php/Terminus/Helpers/Input.php

```diff
@@ -189,6 +186,30 @@ public static function orgname($args, $key) {
   }
 
   /**
+   * Helper function to get role
+   *
+   * @param [array]  $assoc_args Argument array passed from commands
+   * @param [string] $message Prompt to STDOUT
+   * @return [string] $role Name of role
+   */
+  static public function role($assoc_args, $message = 'Select a role for this member') {
+    $roles = array('admin', 'owner');
```

Comment:
:smirk_cat: 

- "Owner" shouldn't be a selectable role
- "developer", "team_member" and "admin" are the only available roles for SiteUserMemberships (OrganizationUserMemberships have some other flavors)